aptos-move/block-executor/src/view.rs

```diff
@@ -161,8 +174,19 @@ impl<'a, T: Transaction, X: Executable> ParallelState<'a, T, X> {
 }
 
 pub(crate) struct SequentialState<'a, T: Transaction, X: Executable> {
-    pub(crate) unsync_map: &'a UnsyncMap<T::Key, T::Value, X>,
-    pub(crate) _counter: &'a u32,
+    unsync_map: &'a UnsyncMap<T::Key, T::Value, X>,
+    unsync_liftings: RefCell<UnsyncLiftings>,
```

Comment:
Why RefCell in SequentialState but not in ParallelState?

Reply:
This is just to hide both the counter and the hashmap under the same refcell. Otherwise would have moved it inside, maybe it's actually cleaner to do so